samples/system-test/topics.test.js

```diff
@@ -170,7 +169,7 @@ describe('topics', () => {
     const actualWait = publishTime.getTime() - startTime;
 
     assert.strictEqual(data.toString(), expectedMessage.data);
-    assert(actualWait >= waitTime);
+    assert(actualWait <= waitTime);
```

Comment:
I'm slightly confused as to why this logic is flipped. Was this incorrect before?

Reply:
When we publish 1 message with a batch setting of 10 maxMessages, the publisher client will try to wait until there's 10 messages until the timeout time is reached first. This causes the actual wait time to be longer.

When we publish 10 messages with a tach setting of 10 maxMessages, the publisher is ready to publish the batch right away, before it hits the timeout time. This causes the actual wait time to be shorter.